### PR TITLE
[pwrmgr] Officially move pwrmgr to D1

### PR DESCRIPTION
**Pull request by tjaychen**
Signed-off-by: Timothy Chen <timothytim@google.com>

### CODE REVIEW

File: hw/ip/pwrmgr/doc/checklist.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+Documentation | [SPEC_COMPLETE][]     | Done        |[PWRMGR Design Spec]({{<relref "hw/ip/pwrmgr/doc" >}})
+Documentation | [CSR_DEFINED][]       | Done        |
+RTL           | [CLKRST_CONNECTED][]  | Done        |
+RTL           | [IP_TOP][]            | Done        |
+RTL           | [IP_INSTANTIABLE][]   | Done        |
+RTL           | [MEM_INSTANCED_80][]  | N/A         |
+RTL           | [FUNC_IMPLEMENTED][]  | Done        |
+RTL           | [ASSERT_KNOWN_ADDED][]| Done        |
+Code Quality  | [LINT_SETUP][]        | Done        |
+Review        | Reviewer(s)           | Not Started |
```
Comment (sjgitty) [inline hw/ip/pwrmgr/doc/checklist.md:23]: I think the PR acts as review and date. Official signing saved for D3. If we haven't made that formal yet, that's the way we're heading.

**Review by sjgitty — APPROVED** [review]
@sriyerg can you give the thumbs up on the specification?

**Review by msfschaffner — APPROVED** [review]

File: hw/ip/pwrmgr/doc/checklist.md (hunk 1 above)
Comment (tjaychen) [reply hw/ip/pwrmgr/doc/checklist.md:23]: ah sounds good.

**Review by sriyerg — APPROVED** [review]
Probably better to have the checklist / milestone changes in its own commit.

**Comment by tjaychen** [comment]
yes good point.
I'll make sure to do that next time.

**Pull request merged**